types/imtype/index.d.ts

```diff
@@ -0,0 +1,15 @@
+// Type definitions for imtype 1.0.0
+// Project: https://github.com/fatelei/imagetype
+
+export function isBMP(buffer: Uint8Array): boolean;
+
+export function isJPG(buffer: Uint8Array): boolean;
+
+export function isPNG(buffer: Uint8Array): boolean;
+
+export function isGIF(buffer: Uint8Array): boolean;
+
+export function isTIF(buffer: Uint8Array): boolean;
+
+
+export as namespace imType;
```

Comment:
https://github.com/DefinitelyTyped/DefinitelyTyped/pull/69110#discussion_r1535992446

The problem here has not been resolved.

```suggestion

```